safe_control_gym/math_and_models/symbolic_systems.py

```diff
@@ -69,7 +69,7 @@ def setup_model(self):
         # Discrete time dynamics.
         self.fd_func = cs.integrator('fd', self.integration_algo, {'x': self.x_sym,
                                                                    'p': self.u_sym,
-                                                                   'ode': self.x_dot}, {'tf': self.dt}
+                                                                   'ode': self.x_dot}, 0, self.dt
```

Comment:
I am interested as why this was changed. It seemed to be working before, although from what I am seeing on the documentation, perhaps the way we were doing it was deprecated? Can you tell me more?

Reply:
Yes, the dict with `tf` was deprecated starting from casadi 3.6.X.
It still works but you get a warning so I updated it.

Reply:
Awesome, thank you!